develop/smart-contracts/connect-to-polkadot.md

```diff
@@ -1,25 +1,23 @@
 ---
-title: Connect to Asset Hub
-description: Explore how to connect to different Asset Hub networks, configure your wallet, and obtain test tokens for developing and testing smart contracts.
+title: Connect to Polkadot
+description: Explore how to connect to Polkadot Hub, configure your wallet, and obtain test tokens for developing and testing smart contracts.
 ---
 
-# Connect to Asset Hub
+# Connect to Polkadot
 
 <div class="button-wrapper">
-    <a href="#" class="md-button connectMetaMask" value="westendAssetHub">Connect to Westend Asset Hub</a>
+    <a href="#" class="md-button connectMetaMask" value="westendAssetHub">Connect to Westend Hub</a>
 </div>
 
-## Networks
+## Networks Details
 
-Developers can leverage Asset Hub across diverse networks, from TestNets to MainNet. This section outlines the network specifications and connection details for each environment.
-
-### Specifications
+Developers can leverage smart contracts across diverse networks, from TestNets to MainNet. This section outlines the network specifications and connection details for each environment.
 
 === "Westend Asset Hub"
 
     Network name
     ```
-    Asset-Hub Westend TestNet
+    Westend Asset Hub
```

Comment:
I'm not sure what sparked this change, but if making this change here we should probably be consistent, like do we need to also make this change in the MetaMask section of this page where it still uses `Asset-Hub Westend TestNet`?

Reply:
This naming situation should probably be double-checked